src/lib.rs

```diff
@@ -2,32 +2,49 @@
 //!
 //! # Reading
 //!
-//! ```rust,ignore
-//! let r = try!(File::open("somefile"));
+//! ```rust, ignore
+//! let r = File::open("./example.txt");
```

Comment:
Missed a `?`

Reply:
Changed to `//! let r = File::open("something")?;`. Good catch.